gazebo_plugins/include/gazebo_plugins/fill_depth.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Open Source Robotics Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAZEBO_PLUGINS__FILL_DEPTH_HPP_
+#define GAZEBO_PLUGINS__FILL_DEPTH_HPP_
+
+#include <cuda.h>
+
+class FillDepth
```

Comment:
Could you document everything in this header? This is especially important since it will be installed.